meta_capstone/BoardTileCell.m

```diff
@@ -32,15 +32,13 @@ - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range r
         [self textViewDidEndEditing:self.inputView];
         return NO;
     }
-    return YES;
+    return textView.text.length + (text.length - range.length) <= 1;
```

Comment:
what is this checking?

Reply:
textView doesn't have a built in character limit, so this a different way to limit the text to have a range of 1 (allowing only one letter in the tile)